Assets/SEE/Layout/NodeLayouts/IncrementalTreeMapLayout.cs

```diff
@@ -351,4 +362,4 @@ public override bool UsesEdgesAndSublayoutNodes()
             return false;
         }
     }
-}
+}
```

Comment:
It looks like there are still missing EOF newlines in some files, at least GitHub displays it like that.